### PR TITLE
Sanitize urls to limit redirect chains

### PR DESCRIPTION
**Pull request by ashleyjelks**
We ran this logic against the articles on staging, and want these checks to be included on any future articles that are created or existing ones that are updated, so committing these changes here.

### CODE REVIEW

**Review by eessex — APPROVED** [review]
👍 Looks good - thanks for the improvement to backfill logging!

File: src/api/apps/articles/model/sanitize.js [hunk 1]
```diff
@@ -19,6 +20,12 @@ export const sanitizeLink = urlString => {
   if (url.hostname.includes(".artsy.net")) {
     url.protocol = "https:"
   }
+  if (url.href.includes("artsy.net/post/")) {
+    url.href = url.href.replace("/post/", "/article/")
+  }
+  if (url.href.includes("/posts")) {
```
Comment (eessex) [inline src/api/apps/articles/model/sanitize.js:26]: This looks like an unsafe replace-- we should use `"artsy.net/posts/"` like above to ensure we don't change non-artsy URLS

**Review by eessex — CHANGES_REQUESTED** [review]
Actually one blocking comment!

File: src/api/apps/articles/model/sanitize.js (hunk 1 above)
Comment (zephraph) [reply src/api/apps/articles/model/sanitize.js:26]:
```suggestion
  if (url.href.includes("artsy.net/posts")) {
```

**Review by eessex — APPROVED** [review]

**Pull request merged**